src/sass/layout/scanning-page/_hero.scss

```diff
@@ -0,0 +1,61 @@
+.container {
```

Comment:
we have container in _container.scss you dont need write style for this 
look at this picture 
<img width="1440" alt="Screenshot 2021-08-14 at 14 07 41" src="https://user-images.githubusercontent.com/74025448/129444406-22055288-1f0b-4c12-bf08-5b6a9df20027.png">